datacapturing/src/main/java/de/cyface/datacapturing/persistence/MeasurementPersistence.java

```diff
@@ -127,9 +109,18 @@ public void storeData(final @NonNull CapturedData data, final long measurementId
         if (threadPool.isShutdown()) {
             return;
         }
+        if (accelerationsFile == null) {
+            accelerationsFile = new AccelerationsFile(context, measurementIdentifier);
+        }
+        if (rotationsFile == null) {
+            rotationsFile = new RotationsFile(context, measurementIdentifier);
+        }
+        if (directionsFile == null) {
+            directionsFile = new DirectionsFile(context, measurementIdentifier);
+        }
 
-        CapturedDataWriter writer = new CapturedDataWriter(data, resolver, authority, measurementIdentifier, callback);
-
+        CapturedDataWriter writer = new CapturedDataWriter(data, accelerationsFile, rotationsFile, directionsFile,
```

Comment:
Good question. Can we test it? Or should we create a spike task to read about this first ? Who should be assigned? Would you mind to create the task because I think you have more knowledge/experience here ...?